packages/optimizer/lib/transformers/OptimizeAmpBind.js

```diff
@@ -19,19 +19,14 @@ const {skipNodeAndChildren} = require('../HtmlDomHelper');
 const {isTemplate} = require('../AmpConstants');
 
 /**
+ * Inserts i-amphtml-binding attributes on any elements which have bindings to allow the runtime to detect amp-bimd elements more efficiently.
```

Comment:
Could be brain fog, but this was really hard to parse for me. I think below reads a clearer. @CrystalOnScript may have thoughts on it?

allow the runtime to detect amp-bimd elements more efficiently by adding i-amphtml-binding attributes elements that have bindings

Reply:
agree, rephrased